src/app/terms/terms.component.html

```diff
@@ -17,7 +17,7 @@
 <div class="container">
     <div class="col-md-3">
         <br><br><br>
-        <a href="#" >
+        <a href="/terms" >
```

Comment:
Isn't this link redundant? Why do we need a terms and conditions link on the terms page? I would suggest removing the a tag. What say?